pom.xml

```diff
@@ -234,6 +234,30 @@
         </annotationProcessorPaths>
       </configuration>
     </plugin>
+    <plugin>
+      <groupId>org.codehaus.mojo</groupId>
+      <artifactId>exec-maven-plugin</artifactId>
+      <version>${org.codehaus.mojo.exec.plugin.version}</version>
+      <executions>
+        <execution>
+          <id>generate-git-version</id>
+          <phase>generate-resources</phase>
+          <goals>
+            <goal>exec</goal>
+          </goals>
```

Comment:
I think this has to be combined with the existing `exec-maven-plugin` definition. I see this in the build logs:
```
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for io.cryostat:cryostat:jar:2.0.0-SNAPSHOT
[WARNING] 'build.plugins.plugin.(groupId:artifactId)' must be unique but found duplicate declaration of plugin org.codehaus.mojo:exec-maven-plugin @ line 285, column 13
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
```

Reply:
Good catch, fixed.